docs/userguide/services.md

```diff
@@ -91,7 +91,7 @@ separate file instead of being embedded in the lines of the program.*
 ### Locales and Services
 
 The interaction between locales and services is fundamental to ICU. Please refer
-to [Locales and Services](./locale/index.md#Locales_and_Services).
+to [Locales and Services](./locale/index.md#locales-and-services).
```

Comment:
Since this is link with an anchor, I think we'll need to remove the `.md` part, otherwise it'll be broken.

Reply:
Done.